client/src/leap/soledad/client/api.py

```diff
@@ -793,6 +831,10 @@ def raw_sqlcipher_query(self, *args, **kw):
         return self._dbpool.runQuery(*args, **kw)
 
 
+def _process_received_docs(sync_hook_plugin, doc_id_list):
+    sync_hook_plugin.process_received_docs(doc_id_list)
```

Comment:
In general I'm in favor of defining as much functions as we can, but I think here it's not needed just for one line and it might be actually clearer the code if the `plugin.process_received_docs(doc_id_list)` is called directly in the `_sync_callback`.